pennylane/circuit_drawer/circuit_drawer.py

```diff
@@ -116,6 +121,14 @@ def __init__(
         self.full_representation_grid = self.operation_representation_grid.copy()
         self.full_representation_grid.append_grid_by_layers(self.observable_representation_grid)
 
+    def make_wire_conversion_dicts(self, raw_operation_grid, raw_observable_grid):
+        all_operators = list(qml.utils._flatten(raw_operation_grid)) + list(qml.utils._flatten(raw_observable_grid))
+        all_wires = [op.wires for op in all_operators if op is not None]
```

Comment:
Just out of curiosity, when does the ``op is None`` case arise?

Reply:
In the grid, places where no operation is taking place are filled with `None`.